[rospy.client][INFO] 2020-11-11 05:17:46,355: init_node, name[/train_7378_1605071866348], pid[7378]
[xmlrpc][INFO] 2020-11-11 05:17:46,356: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-11-11 05:17:46,356: Started XML-RPC server [http://luigi:46763/]
[rospy.init][INFO] 2020-11-11 05:17:46,357: ROS Slave URI: [http://luigi:46763/]
[rospy.impl.masterslave][INFO] 2020-11-11 05:17:46,357: _ready: http://luigi:46763/
[rospy.registration][INFO] 2020-11-11 05:17:46,358: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-11-11 05:17:46,359: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-11-11 05:17:46,457: registered with master
[rospy.rosout][INFO] 2020-11-11 05:17:46,457: initializing /rosout core topic
[rospy.rosout][INFO] 2020-11-11 05:17:46,462: connected to core topic /rosout
[rospy.simtime][INFO] 2020-11-11 05:17:46,463: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-11-11 05:17:46,474: topic[/bev/flow_image] adding connection to [http://192.168.0.201:41967/], count 0
[rospy.internal][INFO] 2020-11-11 05:17:46,488: topic[/bev/occupancy_image] adding connection to [http://192.168.0.201:41967/], count 0
[rospy.internal][INFO] 2020-11-11 05:17:46,490: topic[/odom] adding connection to [http://192.168.0.201:46537/], count 0
[rospy.internal][INFO] 2020-11-11 05:17:46,714: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-11-11 05:17:48,914: signal_shutdown [atexit]
[rospy.internal][INFO] 2020-11-11 05:17:48,918: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-11-11 05:17:48,918: topic[/bev/flow_image] removing connection to http://192.168.0.201:41967/
[rospy.internal][INFO] 2020-11-11 05:17:48,918: topic[/bev/occupancy_image] removing connection to http://192.168.0.201:41967/
[rospy.internal][INFO] 2020-11-11 05:17:48,918: topic[/odom] removing connection to http://192.168.0.201:46537/
[rospy.impl.masterslave][INFO] 2020-11-11 05:17:48,919: atexit
[rospy.internal][WARNING] 2020-11-11 05:17:48,920: Unknown error initiating TCP/IP socket to 192.168.0.201:35777 (http://192.168.0.201:41967/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

